setup.cfg

```diff
@@ -1,2 +1,92 @@
-[wheel]
-universal = 1
+[metadata]
+name = Mopidy-Webhooks
+version = 1.0.0
+url = https://github.com/djmattyg007/mopidy-webhooks
```

Comment:
should this still point at the source repo?

Reply:
It should 😅